notebooks/1.Getting_Data_Ready.ipynb

```diff
@@ -367,11 +384,16 @@
     "    ]\n",
     "}\n",
     "\n",
-    "create_role_response = iam.create_role(\n",
-    "    RoleName = role_name,\n",
-    "    AssumeRolePolicyDocument = json.dumps(assume_role_policy_document)\n",
-    ")\n",
-    "\n",
+    "try:\n",
+    "    create_role_response = iam.create_role(\n",
+    "        RoleName = role_name,\n",
+    "        AssumeRolePolicyDocument = json.dumps(assume_role_policy_document)\n",
+    "    )\n",
+    "    role_arn = create_role_response[\"Role\"][\"Arn\"]\n",
+    "except iam.exceptions.EntityAlreadyExistsException:\n",
+    "    print(\"The role \" + role_name + \" exists, ignore to create it\")\n",
```

Comment:
Do you think the role related stuff should reside in a util function? Don't feel user needs to look at what all goes into creating and attaching the role policy. What do you feel?

Reply:
Current master branch also does the IAM role creation in the notebook. This time I didn't try to change it but just added error handling of EntityAlreadyExistsException.
We can discuss and address your point in the next set of changes.

Reply:
Sounds good. Can move out in the next set of changes.